src/components/BadgeOverlay/styled/component.js

```diff
@@ -0,0 +1,126 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+
+const StyledComponent = attachThemeAttrs(styled.span)`
```

Comment:
In order to do it more semantic, I think we should name it `StyledBadge`, what's do you think?

Reply:
totally agree!